pingdom-checks/clinical-fhir.ping

```diff
@@ -86,3 +86,89 @@ pingdom save-check \
   -a userids_csv="$NO_USERS" \
   -a integrationids_csv="$CLINICAL_FHIR_SLACK_ID"
 
+
+#
+# Production
+#
+
+# Nginx Proxy Health
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-clinical-fhir-health \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/health" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="clinical-fhir" \
+  -a userids_csv="$NO_USERS" \
+
+# Proxy -> DQ Health Check
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-clinical-fhir-dq-health \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/data-query/health" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="clinical-fhir" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$CLINICAL_FHIR_SLACK_ID"
+
+# Proxy -> VFQ Health Check
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-clinical-fhir-vfq-health \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/vista-fhir-query/health" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="clinical-fhir" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$CLINICAL_FHIR_SLACK_ID"
+
+# Proxy -> VFQ Backend Health Check (Charon)
+pingdom save-check \
+  --template https-public-200 \
+  -a name=production-clinical-fhir-vfq-backend-health \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/vista-fhir-query/backend/health" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group="clinical-fhir" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$CLINICAL_FHIR_SLACK_ID"
+
+# Clinical-Fhir Condition Health Check
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-clinical-fhir-condition \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/r4/Condition?patient=1011537977V693883" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$CLINICAL_FHIR_SLACK_ID"
+
+# Clinical-Fhir Observation Health Check
+pingdom save-check \
+  --template fhir-resource \
+  -a name=production-clinical-fhir-observation \
+  -a host=api.va.gov \
+  -a url="/services/clinical-fhir/v0/r4/Observation?patient=1011537977V693883" \
+  -a authorization_token="$HEALTH_APIS_STATIC_ACCESS_TOKEN" \
+  -a userids_csv="$STATUSPAGE_SANDBOX_CLINICAL_FHIR" \
```

Comment:
Do we have a UserId for `STATUSPAGE_PRODUCTION_CLINICAL_FHIR`?

Reply:
not yet, we will but that linkage doesn't exist yet. For now, set this to `$NO_USERS`